pygmt/src/inset.py

```diff
@@ -0,0 +1,121 @@
+"""
+inset - Create inset figures.
+"""
+import contextlib
+
+import numpy as np
+from pygmt.clib import Session
+from pygmt.helpers import (
+    args_in_kwargs,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@contextlib.contextmanager
+@use_alias(
+    D="location",
+    V="verbose",
+    F="box",
+    M="margin",
+)
```

Comment:
```suggestion
@use_alias(D="position", F="box", M="margin", V="verbose")
```
Most other PyGMT modules (e.g. `legend`, `colorbar` and `logo`) use `position` as an alias for `D`. Best to standardize this. You might need to change `location` -> `position` in other places too.